resource_customizations/monitoring.coreos.com/Prometheus/health.lua

```diff
@@ -1,23 +1,30 @@
-local hs={ status = "Progressing", message = "Waiting for initialization" }
+local hs = { status = "Progressing", message = "Waiting for initialization" }
+found_status = false 
 
 if obj.status ~= nil then
   if obj.status.conditions ~= nil then
     for i, condition in ipairs(obj.status.conditions) do
 
-      if condition.type == "Available" and condition.status ~= "True" then
-        if condition.reason == "SomePodsNotReady" then
-          hs.status = "Progressing"
+      if condition.type == "Available" then
+        found_status = true
```

Comment:
`found_status` should be set to true right after the `if obj.status ~= nil then` statement.